lib/eventFields.js

```diff
@@ -6,7 +6,7 @@ import {
   GraphQLSchema,
 } from 'graphql';
 
-import { EventFetch, AllEventsFetch } from './eventFetch.js';
+import { fetchOne, fetchAll } from './eventOrNewsFetch.js';
```

Comment:
I think we should create a single fetch method that can be used across any document type. The Prismmic API itself is composable, so we should be able to create a nicely flexible solution, here.